src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -89,7 +89,23 @@ public CommandResult execute(Model model) throws CommandException {
 
         model.setPerson(personToEdit, editedPerson);
         model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
-        return new CommandResult(String.format(MESSAGE_EDIT_PERSON_SUCCESS, editedPerson));
+
+        // check for warnings
+        boolean isInvalidVisit = editedPerson.hasVisit() && editedPerson.getVisit().get().isOverdue();
+        boolean isInvalidLastVisit = editedPerson.hasLastVisit() && editedPerson.getLastVisit().get().isFuture();
+
+        if (isInvalidLastVisit && isInvalidLastVisit) {
```

Comment:
Is this a wrong conditional?

Reply:
Thanks for pointing it out. Will fix that